.gitattributes

```diff
@@ -5,4 +5,16 @@
 *.jpg binary
 
 #Files we do not want to distribute
-/tests export-ignore
\ No newline at end of file
+/.appveyor            export-ignore
+/.build               export-ignore
+/.ci                  export-ignore
+/.codecov.yml         export-ignore
+/.editorconfig        export-ignore
+/.gitattributes       export-ignore
+/.github              export-ignore
+/.gitignore           export-ignore
+/.php_cs.dist         export-ignore
+/.travis.yml          export-ignore
+/infection.json.dist  export-ignore
+/phpunit.xml.dist     export-ignore
+/tests                export-ignore
```

Comment:
Seeing https://github.com/infection/infection/blob/bd4cd3208186e14e67857a29a59901b76221b739/.editorconfig#L7 I can recommend using an IDE which is equipped with https://editorconfig.org/#download. Alternatively, if you are using PhpStorm, the corresponding setting to ensure a line feed at end of file can be manually enabled:

![screen shot 2018-10-31 at 08 42 12](https://user-images.githubusercontent.com/605483/47773018-f540fb00-dce8-11e8-8436-0b89c9d85a8e.png)

Reply:
Yea I know this plugins and features of PHPStorm, but I think it without a new line at master and I kept it this way.